### PR TITLE
Add python 3.11.4 benchmark results

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
Add benchmark results for updating runner to  Python 3.11.4. 

### How to review 
- Have a look at the results and see if they are okay. Got some fails on the baseline results
- Add whether we should approve or disapprove the merge

### CODE REVIEW

**Comment by petechd** [comment]
[Update Runner packages and version to Python 3.11 (2 SP)](https://trello.com/c/WFO5Lbxs/5725-update-runner-packages-and-version-to-python-311-2-sp)

File: doc/performance-investigations/0023-upgrade-python-to-3.11.4/summary.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Results
+
+Results based on 99th percentile timings.
+| Metric              | Baseline | Investigation |
+| ------------------- | -------- | ------------- |
+| Questionnaire GETs  | 827ms    | 826ms         |
+| Questionnaire POSTs | 565ms    | 539ms         |
+| All requests        | 730ms    | 720ms         |
+| Total Requests      | 61,358   | 61,269        |
+| Total Failures      | 0 .      | 0             |
```
Comment (berroar) [inline doc/performance-investigations/0023-upgrade-python-to-3.11.4/summary.md:28]: Is this correct? I know not many but looks like there were some failures for the investigation?

Comment (petechd) [reply doc/performance-investigations/0023-upgrade-python-to-3.11.4/summary.md:28]: I had some gcp glitches which caused 0.3% errors, re-run a smaller benchmark after to confirm and it was error free but I will run another full scale one now and add the results here.

Comment (petechd) [reply doc/performance-investigations/0023-upgrade-python-to-3.11.4/summary.md]: New investigation benchmark (uninterrupted by gcp glitches) results added now.

File: doc/performance-investigations/0023-upgrade-python-to-3.11.4/summary.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+| Run time | 20m |
+| User wait time minimum | 1s |
+| User wait time maximum | 2s |
+| Clients | 50 |
+| Hatch rate | 50 |
+
+## Results
+
+Results based on 99th percentile timings.
+| Metric              | Baseline | Investigation |
```
Comment (MebinAbraham) [inline doc/performance-investigations/0023-upgrade-python-to-3.11.4/summary.md:22]: Any significant change in CPU/Mem between the two?.

Comment (petechd) [reply doc/performance-investigations/0023-upgrade-python-to-3.11.4/summary.md:22]: Pretty much the same:
Screenshot 2023-09-05 at 09 15 27
Screenshot 2023-09-05 at 09 15 37

**Review by Yuyuutsu — APPROVED** [review]
lgtm !

**Review by rmccar — APPROVED** [review]

**Review by berroar — APPROVED** [review]

**Pull request merged**